### ISSUE TITLE
3.2 and 3.3 support?

### ISSUE THREAD

**Issue by harlowja** [jd/tenacity#4]
Why have these?

```
Programming Language :: Python :: 3.2

Programming Language :: Python :: 3.3
```
Remove them?

**Comment by jd** [comment]
I don't mind. The code currently works on 3.2 and 3.3 and is tested by Travis. I don't use 3.2 nor 3.3 so we could remove them if it bothers us, until then… why not? WDY?T

**Comment by harlowja** [comment]
Less versions of python that I don't have to think about the better :-P

**Comment by jd** [comment]
Closed by #10
